hexvexed-0.0.1.recipe

```diff
@@ -1,6 +1,7 @@
-SUMMARY="A hexagon based game for Haiku"
-DESCRIPTION="This is a hexagon based tile puzzle game forked from DarkWyrm's BeVexed \
-code to support 6 sides instead of four."
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION="This is a great puzzle game for Haiku,Verry addictive game and easy to play.\
+Just put hexagon tiles in the grid on left such that the numbers match whatever 2 tiles touch. \
+This game forked from DaekWyrm's BeVexed but you can arrange 6 sides instread of 4."
 
```

Comment:
My last comment seems to have been lost. At least the errors are still in the latest PR.